src/System.Net.Http/tests/FunctionalTests/PlatformHandlerTest.cs

```diff
@@ -42,6 +42,11 @@ public sealed class PlatformHandler_DiagnosticsTest : DiagnosticsTest
         protected override bool UseSocketsHttpHandler => false;
     }
 
+    public sealed class PlatformHandler_HttpClient_SelectedSites_Test : HttpClient_SelectedSites_Test
+    {
+        protected override bool UseSocketsHttpHandler => false;
+    }
```

Comment:
We're bailing out of the test if UseSocketsHttpHandler is false... do we need this at all then?  Could the test just be moved to a SocketsHttpHandler-specific test class?

Reply:
That is for the "investigating" test case: if it is interesting somehow I typically wanted to see the behavior of the platform handler (and perhaps debug it too).